apps/backend/src/libs/types/service.type.ts

```diff
@@ -2,9 +2,6 @@ type Service<T = unknown> = {
 	create(payload: unknown): Promise<T>;
 	delete(): Promise<boolean>;
 	find(): Promise<T>;
-	findAll(): Promise<{
-		items: T[];
-	}>;
```

Comment:
Should not change this type

Reply:
Does this mean I also need to revert removing it from the backend? Ticket was labelled with backend also so I assume i have to make changes at the backend as well.